common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1238,17 +1238,17 @@ def submit_problem(self, data, override_time=False):
             # the full exception, including traceback,
             # in the response
             if self.runtime.user_is_staff:
-                msg = u"Staff debug info: {tb}".format(tb=cgi.escape(traceback.format_exc()))
+                msg = u"Staff debug info: {tb}".format(tb=traceback.format_exc())
```

Comment:
There's another instance of `cgi.escape(traceback.format_exc())` in the file. Should that occurrence also be changed?

Reply:
I changed it, but I honestly don't know when that code branch is executed -- it's only if DEBUG is on?